samples/system-test/spanner.test.js

```diff
@@ -1530,6 +1535,118 @@ describe('Spanner', () => {
     });
   });
 
+  // create_database_with_proto_descriptor
+  it('should create a database with a proto descriptor', async () => {
+    const output = execSync(
+      `node database-create-with-proto-descriptor.js ${PROTO_DATABASE_ID1} ${DATABASE_ID} ${PROJECT_ID}`
+    );
+    assert.match(
+      output,
+      new RegExp(`Waiting for creation of ${DATABASE_ID} to complete...`)
+    );
+    assert.match(
+      output,
+      new RegExp(`Created database ${DATABASE_ID} with proto column.`)
+    );
+  });
+
+  // database_update_with_proto_descriptor
+  it('should update database schema with a proto descriptor', async () => {
+    const output = execSync(
+      `node database-update-with-proto-descriptor.js ${PROTO_DATABASE_ID2} ${DATABASE_ID} ${PROJECT_ID}`
+    );
+    assert.match(
+      output,
+      new RegExp(`Waiting for update on ${DATABASE_ID} to complete...`)
+    );
+    assert.match(
+      output,
+      new RegExp(`Updated database ${DATABASE_ID} with proto column.`)
+    );
+  });
+
+  describe('proto columns', () => {
+    const protoDatabaseId = `test-proto-database-${CURRENT_TIME}`;
```

Comment:
Why do we need to create new proto database , we already have `PROTO_DATABASE_ID1` and `PROTO_DATABASE_ID2` , can't we reuse these ?

Reply:
@surbhigarg92, Yes we can use it but then we need to assume the tests get executed in the sequential order of their definition. I think we can assume it because other tests are also assuming the same.